sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jdbc/JdbcPropertiesBeanPostProcessor.java

```diff
@@ -61,14 +64,10 @@ public Object postProcessBeforeInitialization(Object bean, String beanName) thro
             }
 
             boolean isPasswordProvided = StringUtils.hasText(dataSourceProperties.getPassword());
-
             if (isPasswordProvided) {
-                if (isAzureHostedDatabaseService(url)) {
-                    LOGGER.info("Azure managed database services with password detected, it is encouraged to use the"
-                        + "credential-free feature. Please refer to https://aka.ms/spring/credentail-free.");
-                } else {
-                    LOGGER.debug("Value of 'spring.datasource.password' is provided, skip enhancing jdbc url.");
-                }
+                LOGGER.info("Value of 'spring.datasource.password' is detected, if you are using Azure hosted services,"
```

Comment:
when this log will be priented out?

Reply:
it should be logged and only when the application can use credential-free but not